docker/cortx-deploy/cortx-data/Dockerfile

```diff
@@ -0,0 +1,79 @@
+#!/bin/bash
```

Comment:
Could you please remove all files related to `cortx-data` from this PR?

Reply:
Not sure why they get added also I am not getting option to sync with remote branch.  @mukul-seagate11  Can you please help.

Reply:
@shailesh-vaidya, looks like you removed these files right?